components/badge/style.ts

```diff
@@ -1,37 +1,48 @@
 import { css } from "emotion";
-import { getColors, getFonts } from "../../shared/style";
+import { getColors, getFonts } from "../../shared/styles/core";
 
-const badgeAppearances = {
+const badgeTheme = {
   "default": css`
     background-color: ${getColors().greyLight};
     border-color: ${getColors().greyLight};
     color: #1b2029;
```

Comment:
Maybe we can use a named color here as well.

Reply:
If this is not a color named from Design we could name this `ebonyClay`. It just feels cleaner to have all colors named.